orchestration/dap_orchestration/resources.py

```diff
@@ -0,0 +1,160 @@
+import os
+import subprocess
+from typing import Any
+from dataclasses import dataclass
+
+from dagster import DagsterLogManager, Array, Field, resource, StringSource, IntSource
+from dagster.core.execution.context.init import InitResourceContext
+
+
+@dataclass
+class DataflowBeamRunner:
+    working_dir: str
+    logger: DagsterLogManager
+    region: str
+    worker_machine_type: str
+    autoscaling_algorithm: str
+    num_workers: int
+    max_num_workers: int
+    google_project: str
+
+    def __post_init__(self) -> None:
+        self.arg_dict = {
+            "region": self.region,
+            "workerMachineType": self.worker_machine_type,
+            "autoscalingAlgorithm": self.autoscaling_algorithm,
+            "numWorkers": str(self.num_workers),
+            "maxNumWorkers": str(self.max_num_workers),
+            "project": self.google_project,
+            "runner": "dataflow",
+            "experiments": "shuffle_mode=service",
+        }
+
+    def run(
```

Comment:
nit: these args can comfortably fit on one line, so they should be formatted as such